cmd/archive/unpack.go

```diff
@@ -88,10 +108,16 @@ func main() {
 		os.Exit(FailedChdirAfterChroot)
 	}
 
-	if err = archive.UnpackNoChroot(op, os.Stdin, filterSpec, "/"); err != nil {
-		op.Error(err)
-		os.Exit(FailedInvokeUnpack)
-	}
+	return op
+}
 
-	os.Exit(Success)
+func main() {
+	switch len(os.Args) {
```

Comment:
I think all these magic numbers need some docs.

Reply:
yeah i just need a better argument structure entirely, this is all implicit and not great

Reply:
at the very least i'll add docs